### DISCUSSION TITLE
Vulnerabilities ideas

### DISCUSSION THREAD

**Discussion by tobyash86** [tobyash86/WebGoat.NET#83] [Ideas]
This is a place where you can post your ideas for potential vulnerabilities in the WebGoat.NET code. Currently, you can help us to create vulnerabilities for OWASP Top 10-2021.

**Comment by brandonceja** [comment]
Hi authors! great job you´ve done here, I was wondering if you have a list of the current vulnerabilities that are present in this application? I'm using your application for analysing the results of different DAST tools, and it would be great if I could compare it with the real intended findings of this app.
